packages/prompts/src/index.ts

```diff
@@ -624,3 +624,41 @@ Only return Node.js code. DO NOT include any English text before or after the No
 
 `;
 };
+
+export const englishStyleGuidePrompt = `# English Style Guide
+
+## Instructions
+
+Your goal is to apply two style guides to any English you find in the code:
```

Comment:
```suggestion
Your goal is to apply the following style guides to any English you find in the code:
```

Reply:
Changed "all of the following rules" to "the following style guides" to clarify the instructions.